_pytest/logging.py

```diff
@@ -356,12 +357,30 @@ class _LiveLoggingStreamHandler(logging.StreamHandler):
     """
     Custom StreamHandler used by the live logging feature: it will write a newline before the first log message
     in each test.
+
+    During live logging we must also explicitly disable stdout/stderr capturing otherwise it will get captured
+    and won't appear in the terminal.
     """
 
+    def __init__(self, terminal_reporter, capture_manager):
+        """
+        :param _pytest.terminal.TerminalReporter terminal_reporter:
+        :param _pytest.capture.CaptureManager capture_manager:
+        """
+        logging.StreamHandler.__init__(self, stream=terminal_reporter)
+        self.capture_manager = capture_manager
+        self._first_record_emitted = False
+
     def emit(self, record):
-        if not getattr(self, '_first_record_emitted', False):
-            self.stream.write('\n')
-            # we might consider adding a header at this point using self.stream.sep('-', 'live log') or something
-            # similar when we improve live logging output
-            self._first_record_emitted = True
-        logging.StreamHandler.emit(self, record)
+        if self.capture_manager is not None:
+            self.capture_manager.suspend_global_capture()
```

Comment:
I hope that this is a cheap operation, is this the case?

Reply:
I wouldn't expect it to be too expensive; for `sys` capturing is a matter of reading from a file the captured contents so far and changing `sys.stdout` back to its original value; for `fd` capturing it is the same as `sys` with the addition of a `os.dup2` call.